src/main/java/org/worldcubeassociation/db/Persons.java

```diff
@@ -38,4 +38,14 @@ public Person findById(String aId) {
         return existingPersons;
     }
 
+    public boolean existNameAndCountry(String aName, String aCountry) {
```

Comment:
Instead of this new method, could you use the `findByNameAndCountry` method instead?